buildlib/tools/zombie_killer.sh

```diff
@@ -1,20 +1,20 @@
-#!/bin/bash -eEx
+#!/bin/bash -eE
 
-# Search for zombie or defunct processes; kill them all
-out=$(ps -eo pid,stat | awk '$2 == "Z|defunct" {print $1}')
+# Search for zombie with high CPU usage; kill them all
+out=$(ps -eo pid,stat,%cpu | grep Z | awk '$3>85.0 {print $1}')
```

Comment:
Why don't we terminate all the zombie processes?

Reply:
Most zombies are harmless, but we may kill them just out of spite.

Reply:
They may occupy resources and slow down the system. If we don't terminate all the zombie processes maybe at least print all the remaining ones in the end (for future debug purposes)?

Reply:
We do kill them all now, changed per your recommendation.